lib/src/page_list_viewport_gestures.dart

```diff
@@ -48,10 +51,11 @@ class PageListViewportGestures extends StatefulWidget {
   /// Any pointers not defined within the set will be ignored.
   final Set<PointerDeviceKind> panAndZoomPointerDevices;
 
-  /// Whether the user should be locked into horizontal or vertical scrolling,
-  /// when the user pans roughly in those directions.
+  /// Whether the user should be locked into horizontal or vertical
+  /// scrolling, when the user pans roughly in those directions.
```

Comment:
Was this changed due to line length? If so, we use 120 character lines. Please revert any such unnecessary format changes in this PR.

I'm going to wait until this is done so that I can see what's actually different in this PR.

Reply:
Took a look at your request. Please see the fresh commit.